### DISCUSSION TITLE
mysqldump: not found

### DISCUSSION THREAD

**Discussion by Opekunov** [spatie/db-dumper#169] [Q&A]
I get this error
`The dump process failed with exitcode 127: Command not found: sh: 1: mysqldump: not found`

Mysql in docker.
Tried adding ->setHost, ->setPort.

Problem not resolved

**Comment by itsrexb** [comment]
You can try running this to get the path of your mysqldump
`$ which mysqldump`

then add this to your **dump_binary_path** entry
```

//config/database.php
'connections' => [
	'mysql' => [
		'driver'    => 'mysql'
		...,
		'dump' => [
		   'dump_binary_path' => '/path/to/the/binary', // only the path, so without `mysqldump` or `pg_dump`
		   'use_single_transaction',
		   'timeout' => 60 * 5, // 5 minute timeout
		   'exclude_tables' => ['table1', 'table2'],
		   'add_extra_option' => '--optionname=optionvalue', // for example '--column-statistics=0'
		]
	],
```
[see ](https://spatie.be/docs/laravel-backup/v8/installation-and-setup#content-dumping-the-database)
